src/index.test.ts

```diff
@@ -1,10 +1,14 @@
 import { deepEqual } from "node:assert/strict";
 import { test } from "node:test";
 
-function greet() {}
+const greet = (name: string) => {
+  return {
+    message: `Welcome, ${name}!`
+  };
+};
 
-test("should greet person", () => {
-  const result = greet("Alek");
+test('should greet person', () => {
+  const result = greet('Carl');
```

Comment:
When doing code assignments at SALT, try to not touch the tests unless the docs say so.
You are free (and encouraged) to add tests though!

A nice test to add would be to say how `greet` works if it gets an empty string.